vue/sbc-common-components/src/components/BaseAddress.vue

```diff
@@ -72,15 +72,13 @@
                     v-model="addressLocal.addressRegion"
                     :items="getCountryRegions(addressCountry)"
                     :rules="[...rules.addressRegion, ...spaceRules]"
-                    :readonly="isSchemaBC()"
```

Comment:
These rules seemed like a good idea at the time, but some users would not use AddressComplete to find an address (or their address was not in AddressComplete). Because these components were readonly, users were not able to enter a province or country and could therefore not enter a valid address, which blocked some of them from completing their form.